demisto_sdk/commands/format/update_generic_json.py

```diff
@@ -99,3 +100,18 @@ def update_id(self, field='name') -> None:
 
         if updated_integration_id_dict:
             self.updated_ids.update(updated_integration_id_dict)
+
+    def remove_spaces_end_of_id_and_name(self):
+        """Updates the id and name of the json to have no spaces on its end
+                """
+        updated_id = {}
+        if self.verbose:
+            click.echo('Updating YML ID and name to be without spaces at the end')
+
+        self.data['name'] = self.data['name'].rstrip()
+
+        if is_uuid(self.data['id']):
+            updated_id[self.data['id']] = self.data['id'].rstrip()
```

Comment:
what are these lines for?

Reply:
When you create content item at the UI, and then download it, id of this content item will be uuid - some hash string. If it happened, we will rename it, and then we should update all other items that uses it that id changed. 

Now when you asked I recognized that this is not necessary here.

Reply:
We don't need it here.